app/src/main/scala/org/alephium/explorer/persistence/schema/CustomGetResult.scala

```diff
@@ -0,0 +1,126 @@
+      new Transaction.Hash(new Hash(ByteString.fromArrayUnsafe(result.nextBytes())))
+
+  implicit lazy val optionTxHashGetResult: GetResult[Option[Transaction.Hash]] =
+    (result: PositionedResult) =>
+      result
+        .nextBytesOption()
+        .map(bytes => new Transaction.Hash(new Hash(ByteString.fromArrayUnsafe(bytes))))
+
+  implicit lazy val optionBlockEntryHashGetResult: GetResult[Option[BlockEntry.Hash]] =
+    (result: PositionedResult) =>
+      result
+        .nextBytesOption()
+        .map(bytes => new BlockEntry.Hash(new BlockHash(ByteString.fromArrayUnsafe(bytes))))
+
+  implicit lazy val timestampGetResult: GetResult[TimeStamp] =
+    (result: PositionedResult) => TimeStamp.unsafe(result.nextLong())
+
+  implicit lazy val optionTimestampGetResult: GetResult[Option[TimeStamp]] =
+    (result: PositionedResult) => result.nextLongOption().map(TimeStamp.unsafe)
+
+  implicit lazy val groupIndexGetResult: GetResult[GroupIndex] =
+    (result: PositionedResult) => GroupIndex.unsafe(result.nextInt())
+
+  implicit lazy val heightGetResult: GetResult[Height] =
+    (result: PositionedResult) => Height.unsafe(result.nextInt())
+
+  implicit lazy val bigIntegerGetResult: GetResult[BigInteger] =
+    (result: PositionedResult) => result.nextBigDecimal().toBigInt.bigInteger
+
+  implicit lazy val bytestringGetResult: GetResult[ByteString] =
+    (result: PositionedResult) => ByteString.fromArrayUnsafe(result.nextBytes())
+
+  implicit lazy val hashGetResult: GetResult[Hash] =
+    (result: PositionedResult) => Hash.unsafe(ByteString.fromArrayUnsafe(result.nextBytes()))
+
+  implicit lazy val addressGetResult: GetResult[Address] =
+    (result: PositionedResult) => Address.unsafe(result.nextString())
+
+  implicit lazy val u256GetResult: GetResult[U256] =
+    (result: PositionedResult) => {
+      U256.unsafe(result.nextBigDecimal().toBigInt.bigInteger)
+    }
+
+  implicit lazy val optionU256GetResult: GetResult[Option[U256]] =
+    (result: PositionedResult) => {
+      result.nextBigDecimalOption().map(bigDecimal => U256.unsafe(bigDecimal.toBigInt.bigInteger))
+    }
+
+  /**
+    * GetResult type for BlockEntryLite
+    *
+    * @note The order in which the query returns the column values matters.
+    *       For example: Getting (`.<<`) `chainTo` before `chainFrom` when
+    *       `chainFrom` is before `chainTo` in the query result would compile
+    *       but would result in incorrect data.
+    */
+  val blockEntryListGetResult: GetResult[BlockEntryLite] =
+    (result: PositionedResult) =>
+      BlockEntryLite(hash      = result.<<,
+                     timestamp = result.<<,
+                     chainFrom = result.<<,
+                     chainTo   = result.<<,
+                     height    = result.<<,
+                     mainChain = result.<<,
+                     hashRate  = result.<<,
+                     txNumber  = result.<<)
+
+  val blockHeaderGetResult: GetResult[BlockHeader] =
```

Comment:
Nit: both `blockEntryListGetResult` and `blockHeaderGetResult` are `val`, so the `lazy val` definitions above can also be `val`?

And It's from our legacy code 😅